src/elements/Box/Box.stories.tsx

```diff
@@ -25,11 +29,58 @@ export const Styled = () => (
 export const RegularViewProps = () => {
   const r = useRef<View>(null)
   return (
-    <Box
-      px={1}
-      backgroundColor="red100"
-      onLayout={(e) => console.log(e.nativeEvent.layout)}
-      ref={r}
-    />
+    <Flex flex={1}>
+      <Box
+        px={1}
+        height={200}
+        width={200}
+        backgroundColor="red100"
+        onLayout={(e) => console.log(e.nativeEvent.layout)}
+        ref={r}
+      />
+    </Flex>
   )
 }
+
+const colors = ["red10", "green10", "yellow10", "devpurple", "red100", "black10"]
+
+export const GapProps = () => (
+  <List style={{ marginHorizontal: 20 }}>
+    <Text>Gap 👇</Text>
+    <Box
+      borderWidth={1}
+      borderColor="black100"
+      rowGap={10}
+      columnGap={10}
+      // gap={10}
```

Comment:
This is how this was supposed to work but it doesn't work as expected here. Created an [expo snack](https://snack.expo.dev/@gkartalis/nervous-green-hummus) and there is not reproducible.

```suggestion
      gap={10}
```

I think that the `css` function messes this but not sure why 🤔

Reply:
figured it out, it was css breaking stuff - turns out we don't need it at all